torch/csrc/jit/tracer.cpp

```diff
@@ -151,9 +151,14 @@ Value* getValueTrace(const IValue& var) {
   } else {
     // If the values are non-tensors, we try to create constants 
     // and bake those constants into the traced graph
-    Value *constant = state->graph->insertConstant(var);
-    recordSourceLocation(constant->node());
-    return constant;
+    auto constant = tryInsertConstant(*state->graph, var);
+    if (constant) {
+      recordSourceLocation(constant.value()->node());
+      return *constant;
+    }
+    std::ostringstream os;
+    os << "Unknown type used in value trace lookup!";
```

Comment:
Can this error do a better job of reporting what value is not supported as a constant? At the very least it should print the value it is trying to make a constant of.